components/header.js

```diff
@@ -73,7 +73,9 @@ export default props => {
         <div className="header__container">
           <nav>
             <div className="nav__logo">
-              <img src="/static/favicons/android-chrome-192x192.png" alt="" />
+              <a href="/">
```

Comment:
Use nextjs Link[https://github.com/zeit/next.js/#with-link] component for this instead of regular anchor tag.

Reply:
Okay, will work on that.